app/schemas.py

```diff
@@ -1,7 +1,25 @@
-from datetime import (
-    datetime,
-    date,
-    timedelta)
+"""
+This file declares marshmallow-sqlalchemy schemas used to serialize and deserialize objects.
+
+https://marshmallow-sqlalchemy.readthedocs.io/en/latest/index.html#generate-marshmallow-schemas
```

Comment:
These seem to be the most substantive changes.  Many of the test failures had to do with model instances no longer being serialized or deserialized the same way as before the upgrades.  A function that expected a model instance would instead get a dictionary, etc.

https://marshmallow-sqlalchemy.readthedocs.io/en/latest/index.html